quill-core/src/main/scala/io/getquill/dsl/DynamicQueryDSL.scala

```diff
@@ -60,7 +64,9 @@ trait DynamicQueryDsl {
   implicit def toQuoted[T <: Action[_]](q: DynamicAction[T]): Quoted[T] = q.q
 
   def dynamicQuery[T](implicit t: ClassTag[T]): DynamicEntityQuery[T] =
-    dynamicQuerySchema(t.runtimeClass.getName.split('.').last.split('$').last)
```

Comment:
The `dynamicQuerySchema` method needs to use `Fixed` naming strategy since it's the equivalent of `querySchema`. Need to have separate invocation of `Entity` here.